diff --git a/index.d.ts b/index.d.ts
index a292b9cc1b626a85ee75021f028213d4c2c56228..35412203c629926dbab3e36c0e7539dc9cd54c88 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -180,6 +180,7 @@ declare module "typewriter-effect" {
   }
 
   const TypewriterComponent: React.FunctionComponent<{
+    component?: React.ElementType
     onInit?: (typewriter: TypewriterClass) => void
     options?: Partial<Options>
   }>